fileupload.go

```diff
@@ -14,7 +14,15 @@ import (
 type FileUploadParams struct {
 	Params
 	Purpose FileUploadPurpose
-	File    *os.File
+
+	// FileData is a reader with the contents of the file that should be uploaded.
+	// Filename is just the name of the file without path information.
+	FileData io.Reader
+	Filename string
+
+	// File is a deprecated form of FileData and Filename that will do the same thing.
+	// Please use FileData and Filename instead.
```

Comment:
Good call documenting the old stuff too. Thanks! Can you tweak this a little bit to read:

> File is a deprecated form of FileData and Filename that will do the same thing, but allows referencing a file directly. Please prefer the use FileData and Filename instead.